basxconnect/core/layouts/editperson.py

```diff
@@ -436,38 +434,31 @@ def revisionstab(request):
 def relationshipstab(request):
     return (
         _("Relationships"),
-        layout.datatable.DataTable.from_model(
-            Relationship,
-            hg.F(
-                lambda c, e: c["object"].relationships_to.all()
-                | c["object"].relationships_from.all()
+        layout.grid.Grid(
+            layout.form.FormsetField.as_datatable(
+                "relationships_to",
+                [
+                    layout.datatable.DataTableColumn("", hg.C("object")),
+                    "type",
+                    "person_b",
+                    "start_date",
+                    "end_date",
+                ],
+                title=_("Relationships from .. to"),
+                can_delete=False,
```

Comment:
why can these not be deleted?

Reply:
Ups, that has been left from testing, thanks.